cmd/earthly/org_cmds.go

```diff
@@ -56,6 +56,37 @@ func (app *earthlyApp) orgCmds() []*cli.Command {
 
 func (app *earthlyApp) orgCmdsPreview() []*cli.Command {
 	return []*cli.Command{
+		{
+			Name:        "members",
+			Aliases:     []string{"member"},
```

Comment:
Nit: For consistency.

```suggestion
			Name:        "member",
			Aliases:     []string{"members"},
```